internal/distro/rhel90/package_sets.go

```diff
@@ -969,9 +968,7 @@ func anacondaPackageSet(t *imageType) rpmmd.PackageSet {
 			"glibc-all-langpacks",
 			"gnome-kiosk",
 			"google-noto-sans-cjk-ttc-fonts",
-			"grub2-efi-x64-cdboot",
```

Comment:
Is this pulled in via "grub2-tools-efi", or did this just get removed? Could maybe use a comment in the commit message.

Reply:
Same story with syslinux above: [anacondaBootPackageSet for x86](https://github.com/achilleas-k/osbuild-composer/blob/836fee7f39541cd544880abcba8c406dba5b580e/internal/distro/rhel90/package_sets.go#L119)